storage/buckets/buckets_test.go

```diff
@@ -471,8 +471,8 @@ func TestPublicAccessPrevention(t *testing.T) {
 		t.Errorf("getPublicAccessPrevention: got %v, want %v", got, want)
 	}
 
-	if err := setPublicAccessPreventionUnspecified(ioutil.Discard, bucketName); err != nil {
-		t.Errorf("setPublicAccessPreventionUnspecified: %v", err)
+	if err := setPublicAccessPreventionInherited(ioutil.Discard, bucketName); err != nil {
```

Comment:
Inherited is already tested below, so you can just delete this.

Reply:
good call